x-pack/legacy/plugins/maps/public/angular/get_initial_layers.d.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export function getInitialLayers(layers?: unknown): unknown[];
```

Comment:
Can you change to `getInitialLayers(layerListJSON: string, initialLayers?: unknown[])`

Reply:
It's called at least once without any params, so I think I have to have the first param be optional.
`const layerList = state && state.layerList ? state.layerList : getInitialLayers();`